examples/dataflow-flex-python/README.md

```diff
@@ -0,0 +1,202 @@
+## Dataflow Python Flex Template
+
+This example contains a sample Dataflow job which reads a XML file and inserts the records to BQ table.
+It explains how to create Flex template and run it in a restricted environment where there is no 
+internet connectivity to dataflow launcher or worker nodes. It also run dataflow template on shared VPC.
+Example also contains a DAG which can be used to trigger Dataflow job from composer. It also demonstrates
+how we can use cloudbuild to implement CI/CD for this dataflow job.
+
+There are two docker files in the repo, the Dockerfile present at the root of directory is to create a image
+with pre-installed dependencies required for workers to run the job. This image is used by worker at job run time to
+avoid dependency downloading, its helpful in scenarios where workers dont have internet connectivity.
+
+Another Dockerfile in directory `df-sample-xml-to-bq` creates an image to be used by launcher in order to launch the
+dataflow job.
+
+
+### Setup
+```
+export PROJECT_ID=<project_id>
+export PROJECT_NUMBER=$(gcloud projects describe $PROJECT_ID --format="value(projectNumber)")
+export HOST_PROJECT_ID=<HOST_PROJECT_ID>
+export INPUT_BUCKET_NAME=pw-df-input-bkt
+export STAGING_BUCKET_NAME=pw-df-temp-bkt
+export LOCATION=us-central1
+export BQ_DATASET=bqtoxmldataset
+export NETWORK=shared-vpc
+export SUBNET=bh-subnet-usc1
```

Comment:
Network and subnetwork and firewall rules are not created in this example, I would mention as prerequisite.

Reply:
done